okta/data_source_okta_apps.go

```diff
@@ -152,7 +152,7 @@ func (d *AppsDataSource) Read(ctx context.Context, req datasource.ReadRequest, r
 	filterValue := strings.Join(filters, " AND ")
 
 	// Read the list of applications from Okta.
-	apiRequest := d.config.oktaSDKClientV5.ApplicationAPI.ListApplications(ctx).Filter(filterValue)
+	apiRequest := d.config.oktaSDKClientV5.ApplicationAPI.ListApplications(ctx).Filter(filterValue).Limit(200)
```

Comment:
please use the `defaultPaginationLimit` const https://github.com/okta/terraform-provider-okta/blob/master/okta/utils.go#L29